lfucg-exactions/server/accounts/serializers.py

```diff
@@ -224,6 +236,7 @@ class Meta:
             'credits_available',
 
             'total_costs',
+            'dollar_values',
```

Comment:
I'm not a huge fan of these dollar values being serialized in addition to the unformatted values above - this basically sends the same information twice, but with two different formats.  It's not a huge deal, but in general it would be better to do the formatting on the front end (or only send the formatted values if you never actually need the unformatted values)

Reply:
I agree, format on the front end. Don't send redundant data.